scratch/es-to-aoss-migration/dump-aoss-lidvids.py

```diff
@@ -0,0 +1,81 @@
+import json
+import os
+
+import requests
+from opensearchpy import RequestsAWSV4SignerAuth, OpenSearch, RequestsHttpConnection
+from requests_aws4auth import AWS4Auth
+
+from pds.registrysweepers.utils.db import query_registry_db_with_search_after
+from pds.registrysweepers.utils.db.client import get_opensearch_client_from_environment, get_aws_aoss_client_from_ssm, \
+    get_aws_credentials_from_ssm
+
+#START VALIDATED CODE
+iam_role_name = 'temp-mcp-ec2-opensearch-role'
+aoss_host = 'b3rqys09xmx9i19yn64i.us-west-2.aoss.amazonaws.com'
+
+credentials = get_aws_credentials_from_ssm(iam_role_name)
+
+auth = RequestsAWSV4SignerAuth(credentials, 'us-west-2', 'aoss')
+
+# ## VALIDATED
```

Comment:
My "reflex" whenever I see commented-out code is to either:

1. Insert a comment explaining why it's commented-out; but if that can't be provided, then…
2. Nuke the commented-out code completely

Which is it going to be here? 😏

Reply:
A grumbling acknowledgement that if "it's just a scratch script" isn't a good enough reason to avoid practices like semantically-clear code and proper use of functions/classes, it isn't a good enough reason to leave it full of frantic development cruft, either.